pkg/apis/core/v1alpha1/types.go

```diff
@@ -250,6 +250,57 @@ const (
 // DataVolumeCloneSourceSubresource is the subresource checked for permission to clone
 const DataVolumeCloneSourceSubresource = "source"
 
+//StorageProfile provides a CDI specific recommendation for storage parameters
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+// +kubebuilder:storageversion
+// +kubebuilder:resource:shortName=sp;sps,scope=Cluster
+type StorageProfile struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   StorageProfileSpec   `json:"spec"`
+	Status StorageProfileStatus `json:"status,omitempty"`
+}
+
+//StorageProfileSpec defines specification for StorageProfile
+type StorageProfileSpec struct {
+	// ClaimPropertySets is a provided set of properties applicable to PVC
+	ClaimPropertySets []string `json:"claimPropertySets,omitempty"`
+}
+
+//StorageProfileStatus provides the most recently observed status of the StorageProfile
+type StorageProfileStatus struct {
+	// The StorageClass name for which capabilities are defined
+	StorageClass *string `json:"storageClass,omitempty"`
+	// The Storage class provisioner plugin name
+	Provisioner *string `json:"provisioner,omitempty"`
+	// ClaimPropertySets computed from the spec and detected in the system
+	ClaimPropertySets []ClaimPropertySet `json:"claimPropertySets,omitempty"`
+}
+
+type ClaimPropertySet struct {
```

Comment:
The travis is failing because the type ClaimPropertySet should have comment
My suggestion is:
// ClaimPropertySet defines the specification of the volume

Reply:
oh yes, thanks for comment, 
it is a copy paste error, as you can see the comment is available in  in v1beta1, and I can see I messed up some other parts in v1beta1, I will fix it, and then I think I can remove [WIP]

Reply:
as I learned now, the alpha1 is no longer needed for something that is a new functionality, so I have now removed the alpha1 CRD, 

also I have corrected the names and comments